src/components/sandbox/Sandbox.tsx

```diff
@@ -22,7 +24,7 @@ type Serializer = (el: any, properties: ComponentBinding[]) => string;
 interface SandboxProps {
   properties?: ComponentBinding[];
   formItemProperties?: ComponentBinding[];
-  note?: string;
+  note?: string | { type?: GoabCalloutType; heading?: string; content: string };
```

Comment:
The shared type appears to work. Looks good to me!

<img width="778" alt="image" src="https://github.com/user-attachments/assets/90f4e28a-cbc5-4b41-b199-bb225acaefcb" />